lab_8_llm/main.py

```diff
@@ -171,6 +245,23 @@ def _infer_batch(self, sample_batch: Sequence[tuple[str, ...]]) -> list[str]:
         Returns:
             list[str]: Model predictions as strings
         """
+        batch_pred_list = []
+
+        for sequence in sample_batch[0]:
+            inputs = self._tokenizer(sequence,
+                                     max_length=self._max_length,
+                                     padding=True,
+                                     truncation=True,
+                                     return_tensors="pt")
+            outputs = self._model(**inputs)
+
+            predicted = torch.nn.functional.softmax(outputs.logits, dim=1)
```

Comment:
Is it possible not to use here `softmax`?

Reply:
implementation has been changed